src/command/lsp/mod.rs

```diff
@@ -121,27 +122,20 @@ async fn run_lsp(client_config: StudioClientConfig, lsp_opts: LspOpts) -> RoverR
                 Config {
                     root_uri: String::from(supergraph_yaml_url.clone()),
                     enable_auto_composition: false,
-                    force_federation: false,
+                    force_federation: true,
```

Comment:
at first blush, this sort of worries me; what's the reason for the change?

Reply:
So this actually relates to the bump of the language server to 0.3.5, as can be seen in this PR here: https://github.com/mdg-private/language-server/pull/237/files.

Essentially setting this option ensures that the language server always treats what's given to it as a supergraph rather than a monograph, because if that doesn't happen you get lots of errors when trying to compose along the lines of "This isn't a supergraph so what do you mean you've tried to compose?!?" only more computery. 

So this is to ensure the language-server always treats the supergraphs that are passed to it properly, no matter if they have 0, 1 or n subgraphs.